From b8c496a666647bfdf2b0bc54979ef06363868e93 Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Mon, 13 Jan 2014 19:26:01 +0000
Subject: [PATCH] Update generated files for new copy-attributes API.

This updates commit efb5f1841524ec75d765368b447270b8a3aff8ac.

(cherry picked from commit e68baf1954f9b68a228c9ae99a738f2d750418c1)
---
 gobject/Makefile.inc | 6 ++++--
 po/POTFILES          | 1 +
 2 files changed, 5 insertions(+), 2 deletions(-)

diff --git a/gobject/Makefile.inc b/gobject/Makefile.inc
index fbc9eb6..7024754 100644
--- a/gobject/Makefile.inc
+++ b/gobject/Makefile.inc
@@ -92,7 +92,8 @@ guestfs_gobject_headers= \
   include/guestfs-gobject/optargs-mke2fs.h \
   include/guestfs-gobject/optargs-mktemp.h \
   include/guestfs-gobject/optargs-syslinux.h \
-  include/guestfs-gobject/optargs-remount.h
+  include/guestfs-gobject/optargs-remount.h \
+  include/guestfs-gobject/optargs-copy_attributes.h
 
 guestfs_gobject_sources= \
   src/session.c \
@@ -166,4 +167,5 @@ guestfs_gobject_sources= \
   src/optargs-mke2fs.c \
   src/optargs-mktemp.c \
   src/optargs-syslinux.c \
-  src/optargs-remount.c
+  src/optargs-remount.c \
+  src/optargs-copy_attributes.c
diff --git a/po/POTFILES b/po/POTFILES
index 1e831ff..76c41ec 100644
--- a/po/POTFILES
+++ b/po/POTFILES
@@ -163,6 +163,7 @@ gobject/src/optargs-btrfs_filesystem_resize.c
 gobject/src/optargs-btrfs_fsck.c
 gobject/src/optargs-compress_device_out.c
 gobject/src/optargs-compress_out.c
+gobject/src/optargs-copy_attributes.c
 gobject/src/optargs-copy_device_to_device.c
 gobject/src/optargs-copy_device_to_file.c
 gobject/src/optargs-copy_file_to_device.c
-- 
1.8.4.2

